build.gradle

```diff
@@ -34,6 +34,10 @@ task coverage(type: JacocoReport) {
             fileTree(dir: it, exclude: ['**/*.jar'])
         })
     }
+    tasks.withType(Test) {
+        scanForTestClasses = false
+        include "**/*Test*.class"
+    }
```

Comment:
why was this added?

Reply:
Tried this to resolve the CI workflow error but it didnt work, so I will remove it.